cli/lib.rs

```diff
@@ -408,6 +409,23 @@ async fn fmt_command(files: Option<Vec<String>>, check: bool) {
   fmt::format_files(files, check);
 }
 
+async fn test_command(
+  flags: DenoFlags,
+  include: Option<Vec<String>>,
+  fail_fast: bool,
+  quiet: bool,
+) {
+  if let Some(test_file_path) =
+    test_runner::run_test_modules(include, fail_fast, quiet)
+  {
+    let mut flags = flags.clone();
+    flags
+      .argv
+      .push(test_file_path.to_string_lossy().to_string());
+    run_script(flags, test_file_path.to_string_lossy().to_string()).await;
```

Comment:
Shouldn't this be in `test_runner::run_test_modules()`? Also is `.deno.test.ts` being deleted?